app/messages/queries/getMessage.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { AuthorizationError, NotFoundError } from "blitz"
+import db, { Message, User } from "db"
+import { getSession } from "test/utils"
+import getMessage from "./getMessage"
+import { getMessageAttributes } from "test/factories"
+
+jest.mock("app/channels/lib/getUserChannels")
+
+import getUserChannels from "app/channels/lib/getUserChannels" // eslint-disable-line
+
+const mockGetUserChannels = getUserChannels as jest.MockedFunction<typeof getUserChannels>
+
+beforeAll(async () => {
+  await db.message.deleteMany({})
+  await db.user.deleteMany({})
+})
+
+afterAll(async () => {
+  await db.$disconnect()
+})
+
+describe("getMessage", () => {
+  describe("when user is not authorized", () => {
+    it("throws an AuhtorizationError", async () => {
+      const message = (await db.message.create(getMessageAttributes())) as Message & {
+        user: User | null
+      }
+      const criteria = { where: { id: message.id } }
+      try {
+        await getMessage(criteria, getSession())
```

Comment:
You have been using `fail()` in case the function doesn't throw an error on the "throws an error" `it` blocks. Is there any reason for not doing it here?

Reply:
No one, I missed it